lib/Default/MySqlDatabase.class.php

```diff
@@ -1,37 +1,28 @@
 <?php declare(strict_types=1);
-require_once(CONFIG . 'SmrMySqlSecrets.inc');
 
 abstract class MySqlDatabase {
-	// add configuration static members via traits
-	use SmrMySqlSecrets;
-
 	protected static $dbConn;
 	protected static $selectedDbName;
 	protected $dbResult = null;
 	protected $dbRecord = null;
 
-	public function __construct($dbName) {
+	public function __construct($dbName=null, MysqlProperties $mysqlProperties=null) {
```

Comment:
This is an iterative step towards setting up IOC Dependency injection with [PHP-DI](https://php-di.org/doc/understanding-di.html) in order to make classes more testable.